src/components/VTabs/mixins/tabs-touch.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Tabs touch
+ *
+ * @mixin
+ */
+export default {
+  methods: {
+    newOffset (direction) {
+      const capitalize = `${direction.charAt(0).toUpperCase()}${direction.slice(1)}`
+      const container = this.$refs.container
+      const items = container.children
+
+      return this[`newOffset${capitalize}`](container, items)
+    },
+    newOffsetPrepend (container, items, offset = 0) {
+      for (let index = this.itemOffset - 1; index >= 0; index--) {
+        const newOffset = offset + items[index].clientWidth
+        if (newOffset >= container.clientWidth) {
+          return { offset: this.scrollOffset - offset, index: index + 1 }
+        }
+        offset = newOffset
+      }
+
+      return { offset: 0, index: 0 }
+    },
+    newOffsetAppend (container, items, offset = this.scrollOffset) {
```

Comment:
This gets called with integers, but then tries to access properties on them as if they were DOM nodes?